mytk/labels.py

```diff
@@ -7,14 +7,16 @@
 class Label(Base):
     def __init__(self, text=None, wrapping=False, **kwargs):
         Base.__init__(self)
-        self.text = text
+        self._initial_text = text
         self.wrapping = wrapping
         self.kwargs = kwargs
-
+        self.text = None
```

Comment:
Since we replaced the getter/setter with this, could we just assign 'text' from the parameters to this and skip the private _initial_text variable?

Reply:
Good point. Fixed.